aait/mobile/group-1/blog_app/lib/features/authentication_and_authorization/data/models/login_user_model.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:blog_app/features/authentication_and_authorization/domain/entities/login_user_entitiy.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+class LoginUserModel extends LoginUserEnity {
+  const LoginUserModel(
+      {
+      required email,
+      required password,
+})
+      : super(
+
+            password: password,
+
+            email: email);
+
+  factory LoginUserModel.fromJson(Map<String, dynamic> json) {
+    return LoginUserModel(
+
+        email: json['email'],
+        password: json['password'],
+);
+  }
+
+  Map<String, dynamic> toJson() {
+    return {
+      "email": email,
+      "password": password,
+
+    };
+  }
+}
+
+
+// import 'package:equatable/equatable.dart';
```

Comment:
**Suggestion** : It is better if the commented code is removed since  it will help to make   our code cleaner and decrease the size of the codebase .